components/webapps/blacklist-whitelist-service/src/main/java/com/wso2telco/services/bw/Queries.java

```diff
@@ -418,12 +315,13 @@ public Response getAllSubscribers() throws SQLException, BusinessException {
 
 	@GET
 	@Path("/apps/{userId}/{operatorId}")
-	@Produces("application/json")
+	@Produces(MediaType.APPLICATION_JSON)
 	public Response getAllApplicationsByUser(@PathParam("userId") String userId,@PathParam("operatorId") String operatorId) throws SQLException, BusinessException {
 
 		//Gson gson = new GsonBuilder().serializeNulls().create();
 		//Id userID = gson.fromJson(jsonBody, Id.class);
 		String jsonString;
+		userId = replaceAllWithWildcard(userId);
 		if (operatorId.equals("_ALL_")) {
```

Comment:
use a constant variable instead of placing "_ALL_". The variable used in multiple places